README.md

```diff
@@ -18,49 +102,60 @@ Variables without a default value and not marked optional, *MUST* be defined for
 | Environment Variable                        | Default                   | Description
 | --------------------------------------------|---------------------------|-------------
 | GPC_CONSUMER_SERVER_PORT                    | 8090                      | The port on which the GPC Consumer Adaptor will run.
+| GPC_CONSUMER_URL                            | http://localhost:8090                                          | The scheme, host, and port of the GP Connect Consumer Adaptor. This is the address GP Connect Consumers use to make requests to the adaptor.
 | GPC_CONSUMER_ROOT_LOGGING_LEVEL             | WARN                      | The logging level applied to the entire application (including third-party dependencies).
 | GPC_CONSUMER_LOGGING_LEVEL                  | INFO                      | The logging level applied to GPC Consumer Adaptor components.
 | GPC_CONSUMER_LOGGING_FORMAT                 | (*)                       | Defines how to format log events on stdout
 
-### Cloud Gateway Configuration Options
+Logging levels are ane of: DEBUG, INFO, WARN, ERROR
```

Comment:
What is ane of? Is it a typo? Is it supposed to be any of?

Reply:
Or a scots "one".

_"Ah, an ahl ah **ane** angin peh an ah"_